File: src/test/java/de/akquinet/ccsp/floodwatch/datamodel/MeasurementMother.java
package de.akquinet.ccsp.floodwatch.datamodel;

import java.util.Date;

public final class MeasurementMother {

    private MeasurementMother() {
    }

    public static Measurement create(Location location, MeasurementImport measurementImport) {
        Measurement measurement = new Measurement(measurementImport, location, new Date(0));
        measurement.setThroughFlowGroup("mali pretok");
        measurement.setThroughFlowInCubicMetersPerSecond(1.03f);
        measurement.setWaterLevelGroup("srednji vodostaj");
        measurement.setWaterLevelInCm(240);
        measurement.setWaterTemperatureInCelsius(16.5f);
        return measurement;
    }
}

File: src/test/kotlin/de/akquinet/ccsp/floodwatch/datamodel/MeasurementMother.kt
package de.akquinet.ccsp.floodwatch.datamodel

import java.util.*

object MeasurementMother {

    fun create(location: Location, measurementImport: MeasurementImport): Measurement {
        val measurement = Measurement(measurementImport, location, Date(0))
        measurement.throughFlowGroup = "mali pretok"
        measurement.throughFlowInCubicMetersPerSecond = 1.03f
        measurement.waterLevelGroup = "srednji vodostaj"
        measurement.waterLevelInCm = 240
        measurement.waterTemperatureInCelsius = 16.5f
        return measurement
    }
}